app/components/user-aggregate-query.js

```diff
@@ -7,7 +7,7 @@ export default Ember.Component.extend({
 	interval: null,
 	groupBy: ['userId'],
 	selectedMetrics: [],
-	userMetrics: ['tSystemPresence', 'tOrganizationPresence', 'tAgentRoutingStatus'],
+	// userMetrics: ['tSystemPresence', 'tOrganizationPresence', 'tAgentRoutingStatus'],
```

Comment:
Is this being populated elsewhere now? If so, the commented code should be removed.